wdtk-client/src/main/java/org/wikidata/wdtk/client/RdfSerializationAction.java

```diff
@@ -251,4 +254,21 @@ private void printHelp() {
 					+ TASK_HELP.get(supportedTask), 75, "\n   ", true));
 		}
 	}
+
+	@Override
+	public String getReport() {
+		String message = "Finished serialization of "
+				+ this.serializer.getTripleCount() + " RDF triples in file "
+				+ this.outputDestination;
```

Comment:
Related to my comment on the JSON test: this should not use this.outputDestination but the same file name as is really used. This is computed in the method createRdfSerializer above. This computation should be moved to another function so that it can be fetched from several places (and this function should call insertDumpInformation to replace the placeholders).